prow/config/config.go

```diff
@@ -441,6 +449,21 @@ type Controller struct {
 	AllowCancellations *bool `json:"allow_cancellations,omitempty"`
 }
 
+// GetDefaultReportTemplate returns the template that belong to a specific repository.
+// If the repository doesn't exist in the report_templates configuration it will
+// inherit the values from its organization, otherwise the default values will be used.
+func (c *Controller) GetDefaultReportTemplate(org, repo string) *template.Template {
+	def := c.ReportTemplates["*"]
```

Comment:
There is no need for this variable to exist, you can just `return c.Reporttemplates["*"]` at the end.

Also, I am wondering if templates are threadsafe? Not a problem introduced by this PR though.